build/Dockerfile

```diff
@@ -4,3 +4,7 @@ COPY group_vars/ ${HOME}/group_vars/
 COPY roles/ ${HOME}/roles/
 COPY watches.yaml ${HOME}/watches.yaml
 COPY playbook.yml ${HOME}/playbook.yml
+
+USER root
+RUN yum install -y redis
+USER 1001
```

Comment:
I don't have much understanding of what's going on here, but still curious why USER 1001?